CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## XX.XX.X
 * Added "SetID(string newDeviceId)" function for managing device id changes.
+* Deprecated "RecordOpenViewAsync" method in ViewCountlyService class.
+* Deprecated "RecordCloseViewAsync" method in ViewCountlyService class.
+* Deprecated "ReportActionAsync" method in ViewCountlyService class.
+* Added new view tracking functionality for replacing the deprecated methods.
```

Comment:
Good luck finding those? please list them here. And above when you deprecate sth pls mention the new alternatives or that there is no alternative